blog/2024-01-10-open-planning.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Open Source Planning for Tractus-X
```

Comment:
I would just call it "Tractus-X open planning". Should we start adding numbering (like PI17) or for which release version is the planning?

Reply:
Already added it to the newtitle section, but forgot it on the title -> done